### DISCUSSION TITLE
"Class \"App\\Models\\Role\" not found"

### DISCUSSION THREAD

**Discussion by bashiirmahad** [spatie/laravel-permission#2390] [Q&A] [answered]
I want to assign role to user but getting this error  "message": "Class \"App\\Models\\Role\" not found",

**Comment by parallels999** [comment]
https://spatie.be/docs/laravel-permission/v5/basic-usage/new-app

**Comment by drbyte** [comment, marked as answer]
In object-oriented programming such as Laravel, anytime you are referring to a model you need to either have/create that model in the current namespace, or specify the namespace of the model which you are calling. In your situation you probably meant to use `\Spatie\Permission\Models\Role`
